test/views/calculation/gain/WorthWhenSoldForLessViewSpec.scala

```diff
@@ -91,14 +91,6 @@ class WorthWhenSoldForLessViewSpec extends CommonPlaySpec with WithCommonFakeApp
 
           lazy val label = doc.select("label")
 
-          s"has the text ${messages.question}" in {
```

Comment:
we should update this test to use the correct selector instead of delete it - we still want to test the label content!

Reply:
fixed on [ad05c19](https://github.com/hmrc/cgt-calculator-resident-shares-frontend/pull/184/commits/ad05c195e7c914ece2526ab81468c71cd7e46ec5)